Gemfile

```diff
@@ -24,7 +24,6 @@ group :test do
   gem 'faker'
   gem 'launchy'
   gem 'selenium-webdriver'
-  gem 'timecop'
```

Comment:
not needed because Rails 5.2 has the function, now, for freezing and travelling in time within a test